static/app/views/alerts/create.tsx

```diff
@@ -96,12 +96,15 @@ class Create extends Component<Props, State> {
   componentDidMount() {
     const {organization, project} = this.props;
 
+    const hasAlertWizardV3 = organization.features.includes('alert-wizard-v3');
```

Comment:
Why repeating code when we create a function from what we have on Line#58 and reuse in both `getInitialState` and `componentDidMount`.

e.g
```suggestion
    const hasAlertWizardV3 = handleAlertWizardV3(organization);
```